components/PrimaryButton.js

```diff
@@ -10,7 +10,7 @@ export default class PrimaryButton extends React.Component {
 
   render() {
     return (
-         <View style={[styles.buttonContainer, {backgroundColor: this.props.backgroundColor}, {height: this.props.height}]}>
+         <View style={[styles.buttonContainer, {backgroundColor: this.props.backgroundColor}, {height: this.props.height}, {width: this.props.width}]}>
```

Comment:
You shouldn't have to define the `width` of the `PrimaryButton` if you want to alter the width, I would go with `margin` so that it's more dynamic